packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -775,16 +776,17 @@ export default class CosmeticFilter implements IFilter {
     return { name: parts[0], args };
   }
 
-  public getScript(js: Map<string, string>): string | undefined {
+  public getScript(scriptlets: Map<string, ScriptletResource>): string | undefined {
```

Comment:
not sure if we need any changes to this function. We can hide the implementation details behind the `scriptlets` (formerly `js`), for example:
* `js.get` can handle aliases
* `js.get` can return `content`